### DISCUSSION TITLE
One theme for multiple frameworks (i.e. Angular and React)

### DISCUSSION THREAD

**Discussion by n8urm8** [primefaces/community#755] [PrimeNG] [answered]
Hello,

I'm trying to figure out if I can create one theme and use it for both frameworks. I've been looking at the theme variables and they seem to be pretty much the same. Is this as simple as it seems? I'm going to do some testing, but if anyone has suggestions/guidance, I'd appreciate it. 

Thank you,

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's not advised but we use almost identical styling for the Prime UI libraries. You can probably do it but be cautious there might be unwanted behaviors.

**Reply by n8urm8** [reply]
Thank you
